### PR TITLE
Add rotation Argument

### PR DESCRIPTION
**Pull request by Simon-Steinmann**
Adds the option to parse the rotation of the PROTO file

### CODE REVIEW

File: demo.py [hunk 1]
```diff
@@ -23,7 +23,9 @@
                      help='If set, the base link will have the option to be static (disable physics)')
 optParser.add_option('--tool-slot', dest='toolSlot', default=None,
                      help='Specify the link that you want to add a tool slot to (exact link name from urdf)')
+optParser.add_option('--rotation', dest='itiRotation', default='0 1 0 0',
+                        help='Set the rotation field of your PROTO file.)')
 options, args = optParser.parse_args()
 
 convert2urdf(options.inFile, options.outFile, options.normal, options.boxCollision, options.disableMeshOptimization,
-             options.enableMultiFile, options.staticBase, options.toolSlot)
+                options.enableMultiFile, options.staticBase, options.toolSlot, options.itiRotation)
```
Comment (DavidMansolino) [inline demo.py:31]:
```suggestion
             options.enableMultiFile, options.staticBase, options.toolSlot, options.itiRotation)
```

File: demo.py [hunk 2]
```diff
@@ -23,7 +23,9 @@
                      help='If set, the base link will have the option to be static (disable physics)')
 optParser.add_option('--tool-slot', dest='toolSlot', default=None,
                      help='Specify the link that you want to add a tool slot to (exact link name from urdf)')
+optParser.add_option('--rotation', dest='itiRotation', default='0 1 0 0',
+                        help='Set the rotation field of your PROTO file.)')
```
Comment (DavidMansolino) [inline demo.py:27]:
```suggestion
                     help='Set the rotation field of your PROTO file.)')
```

File: urdf2webots/importer.py [hunk 3]
```diff
@@ -47,14 +47,17 @@ def mkdirSafe(directory):
 
 
 def convert2urdf(inFile, outFile=None, normal=False, boxCollision=False,
-                 disableMeshOptimization=False, enableMultiFile=False, staticBase=False, toolSlot=None):
+                 disableMeshOptimization=False, enableMultiFile=False, staticBase=False, toolSlot=None, initRotation='0 1 0 0'):
     if not os.path.exists(inFile):
         sys.exit('Input file "%s" does not exists.' % inFile)
+    if not type(initRotation)==str or len(initRotation.split())!=4:
```
Comment (DavidMansolino) [inline urdf2webots/importer.py:53]:
```suggestion
    if not type(initRotation) == str or len(initRotation.split()) != 4:
```

File: urdf2webots/writeProto.py [hunk 4]
```diff
@@ -56,7 +57,7 @@ def declaration(proto, robotName):
     """Prototype declaration."""
     proto.write('PROTO ' + robotName + ' [\n')
     proto.write('  field  SFVec3f     translation     0 0 0\n')
-    proto.write('  field  SFRotation  rotation        0 1 0 0\n')
+    proto.write('  field  SFRotation  rotation        ' + initRotation + '\n')
```
Comment (DavidMansolino) [inline urdf2webots/writeProto.py:60]: I would rather pass this as an argument of the `declaration` function instead of a local variable.

Comment (Simon-Steinmann) [reply urdf2webots/writeProto.py:60]: sure, go ahead. This was the quickest solution I came up with. A cleaner one is of course better

Comment (Simon-Steinmann) [reply urdf2webots/writeProto.py:60]: I can do it too if you want

Comment (DavidMansolino) [reply urdf2webots/writeProto.py:60]: As you prefeer, I can do it but tomorrow morning,

Comment (Simon-Steinmann) [reply urdf2webots/writeProto.py:59]: done

**Comment by omichel** [comment]
The tutorial should probably be updated as well. The list of arguments has one more entry and the rotation step is not needed any more 😉.

**Comment by DavidMansolino** [comment]
Yes, and it should probably be avoided to duplicate the argument list both in the tutorial and README...

**Pull request merged**